cli/cmd/options.go

```diff
@@ -624,6 +623,12 @@ func validateProxyValues(values *l5dcharts.Values) error {
 		}
 	}
 
+	if values.Proxy.DefaultInboundPolicy != "" {
+		if err := validatePolicy(values.Proxy.DefaultInboundPolicy); err != nil {
+			return err
+		}
+	}
```

Comment:
We'll always want to validate `values.Proxy.DefaultInboundPolicy`. With the `values.Proxy.DefaultInboundPolicy != ""` check, we're allowing users to do `linkerd install --set proxy.DefaultInboundPolicy=""` which isn't right.

```suggestion
if err := validatePolicy(values.Proxy.DefaultInboundPolicy); err != nil {
	return err
}
```

Reply:
thanks, good catch --- i think the outer `if` was left over from when we still allowed setting both values.